### PR TITLE
set github action to test prod image

### LINKED ISSUE #456: Explorer lacks test for deployment docker image
the #447 pr passed github ci tests but the deployment image is erroring due to ```Error: No module named 'sentry_sdk'```

There needs to be a ci test for deployment image

### PR DESCRIPTION
**Pull request by pindge**
Fixes: #456

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -110,28 +110,28 @@ up-prod: ## Start using the prod Docker image
 		up -d
 
 init-odc: ## Initialise ODC Database
-	docker-compose exec -T explorer /bin/sh -c \
-		"datacube system init"
+	docker-compose run -T explorer \
```
Comment (alexgleith) [inline Makefile]: `run` does something different to `exec`

You want to use `exec`, as it executes something in the running container. Run creates a new container.

File: .github/workflows/deployment_test.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+name: Deployment Docker Image Tests
```
Comment (alexgleith) [inline .github/workflows/deployment_test.yaml]:
```suggestion
name: Docker Image
```

**Review by alexgleith — APPROVED** [review]
Looks good to me.

One suggestion about the name of the workflow.

And a suggestion for later maybe about caching the build, which will speed it up. 3 minutes isn't too bad though.

**Review by SpacemanPaul — APPROVED** [review]
LGTM

**Pull request merged**